include/abb_librws/rws_interface.h

```diff
@@ -410,6 +425,21 @@ class RWSInterface
    */
   bool setIOSignal(const std::string iosignal, const std::string value);
   
+  /**
+   * \brief A method for setting the data of an RAPID symbol.
+   *
+   * \param task for the name of the RAPID task containing the RAPID symbol.
+   * \param module for the name of the RAPID module containing the RAPID symbol.
+   * \param name for the name of the RAPID symbol.
+   * \param data containing the RAPID symbol's new data.
+   *
+   * \return bool indicating if the communication was successful or not.
```

Comment:
If the method returns true, then the value should have been updated, but it might e.g. depend on the RAPID program if it was accepted or not. A follow up getRAPIDSymbolData(...) call can for example be made to check if the value has been changed or not.

Reply:
Would this be nice/good to add to the comment on the method here? This can be confusing otherwise (`true` only means: no errors while communicating with the controller, it doesn't seem to say anything about whether the operation itself (ie: updating a value) actually succeeded (if I understand you correctly)).

Reply:
Yes, it would, and there are more places that would benefit from clarification of what the return value mean. I will open an issue for it.